bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/embeddable/YouTubeImpl.java

```diff
@@ -84,6 +84,10 @@
     @Nullable
```

Comment:
Shouldn't we introduce a new v2 implementation at `java/com/adobe/cq/wcm/core/components/internal/models/v2/embeddable/YouTubeImpl.java` which extends `v1/embeddable/YouTubeImpl.java`?